views/export1.jade

```diff
@@ -0,0 +1,30 @@
+extends layout
+
+block content
+    include topnav
+    h1 Step 1: Select Translations
+    #export1Table
+
+    a.btn.btn-primary(href='/user/playlist/#{playlistName}/export2') Next
+
+    script(type='text/javascript').
+        var playlistID = !{JSON.stringify(playlistID).replace(/<\//g, "<\\/")}
```

Comment:
take this variable cleanup code and put it into its own function which you can call.